### DISCUSSION TITLE
Can see only Ossec logs in the Alerts

### DISCUSSION THREAD

**Discussion by netonionwest** [Security-Onion-Solutions/securityonion#4476] [Unsupported Versions]
Hi guys,
I'm running Security Onion 2 evaluation and I'm only getting ossec alerts. 
I'm very new to SO but from the videos I've watched I thought I should be seeing things like suricata alerts etc.
When running tcpdump on the sniffing interface I see packets as they should so it shouldn't be a sniffing problem.

Is there anybody who could kindly help me to 'activate' or troubleshoot the other modules?

Attaching suricata logs below :)

`
XXXX@hq-onion-01:~$ cat /opt/so/log/suricata/suricata.log
10/6/2021 -- 07:37:21 - <Notice> - This is Suricata version 6.0.2 RELEASE running in SYSTEM mode
10/6/2021 -- 07:37:21 - <Info> - CPUs/cores online: 32
10/6/2021 -- 07:37:21 - <Info> - dropped the caps for main thread
10/6/2021 -- 07:37:21 - <Info> - eve-log output device (regular) initialized: /nsm/eve-%Y-%m-%d-%H:%M.json
10/6/2021 -- 07:37:21 - <Info> - stats output device (regular) initialized: stats.log
10/6/2021 -- 07:37:21 - <Info> - Running in live mode, activating unix socket
10/6/2021 -- 07:37:23 - <Info> - 1 rule files processed. 22469 rules successfully loaded, 0 rules failed
10/6/2021 -- 07:37:23 - <Info> - Threshold config parsed: 0 rule(s) found
10/6/2021 -- 07:37:23 - <Info> - 22472 signatures processed. 1290 are IP-only rules, 3887 are inspecting packet payload, 17252 inspect application layer, 0 are decoder event only
10/6/2021 -- 07:37:29 - <Info> - can't get cpu-affinity node
10/6/2021 -- 07:37:29 - <Error> - [ERRCODE: SC_ERR_INVALID_NUMERIC_VALUE(60)] - Invalid numeric value
10/6/2021 -- 07:37:29 - <Warning> - [ERRCODE: SC_ERR_INVALID_VALUE(130)] - Invalid number of threads, resetting to default
10/6/2021 -- 07:37:29 - <Info> - Going to use 32 thread(s)
10/6/2021 -- 07:37:30 - <Info> - Running in live mode, activating unix socket
10/6/2021 -- 07:37:30 - <Info> - Using unix socket file '/var/run/suricata/suricata-command.socket'
10/6/2021 -- 07:37:30 - <Notice> - all 32 packet processing threads, 4 management threads initialized, engine started.
10/6/2021 -- 07:37:32 - <Info> - All AFP capture threads are running.
`

**Comment by netonionwest** [comment]
Update, in Hunt it looks like only broadcast traffic is visible (172,16.2.23 is SO). Yet tcpdump reports fine.
Could it be an issue with stenographer?

This is one of the logs for stenographer. This is one of an almost endless message steak (all similar)
```
/usr/bin/stenotype() [0x4063a4]
/lib64/libstdc++.so.6(+0xb5330) [0x7f56b7edf330]
/lib64/libpthread.so.0(+0x7ea5) [0x7f56b8580ea5]
/lib64/libc.so.6(clone+0x6d) [0x7f56b76429fd]
2021/06/11 10:48:26 Stenotype stopped after 2m1.119953076s: stenotype wait failed: signal: aborted (core dumped)
2021/06/11 10:48:26 Deleted stale output file "/nsm/pcap/.1623408446850583"
2021-06-11T10:48:26.379177Z T:778897 [stenotype.cc:567] Starting, page size is 4096
2021-06-11T10:48:26.379295Z T:778897 [stenotype.cc:594] Setting up AF_PACKET sockets for packet reading
2021-06-11T10:48:26.909288Z T:778897 [stenotype.cc:262] Dropping privileges
2021-06-11T10:48:26.909344Z T:778897 [stenotype.cc:267] Dropping priviledges from 941 to GID stenographer
2021-06-11T10:48:26.909742Z T:778897 [stenotype.cc:279] Dropping priviledges from 941 to UID stenographer
2021-06-11T10:48:26.911747Z T:f53b17 [stenotype.cc:466] Thread 0 starting to process packets
2021-06-11T10:48:26.912026Z T:f53b17 [aio.cc:190] Opening packet file /tmp/stenographer855370120/PKT0/.1623408506911929: 4
2021-06-11T10:49:27.966826Z T:f53b17 [aio.cc:122] Closing /tmp/stenographer855370120/PKT0/.1623408506911929 (4), truncating to 0MB and moving to /tmp/stenographer855370120/PKT0/1623408506911929
2021-06-11T10:49:27.967125Z T:f53b17 [aio.cc:190] Opening packet file /tmp/stenographer855370120/PKT0/.1623408567966804: 4
2021-06-11T10:49:27.967705Z T:f4bb07 [index.cc:269] Wrote all index files for /tmp/stenographer855370120/IDX0/.1623408506911929, moving to /tmp/stenographer855370120/IDX0/1623408506911929
```

Any help is appreciated :)

**Comment by weslambert** [comment]
Although you may be seeing packets, it may not necessarily mean you are seeing the traffic you expect.

Do you see different types of Zeek protocol logs in `/nsm/zeek/logs/current` (like HTTP, SSL, DNS, etc)?

Do you see alerts generated in `/nsm/suricata/eve-$TIMESTAMP.json`?

If you perform the following command, do you see indices for `so-zeek-DATE` or `so-ids-$DATE`?

**Comment by netonionwest** [comment]
Hi,

This is what we see in /nsm/zeek/logs/current

drwxr-xr-x 28 zeek socore   4096 Jun 15 01:00 ./
drwxr-xr-x  5 zeek socore   4096 May 18 15:36 ../
drwxr-xr-x  2 zeek zeek     4096 May 31 14:34 2021-05-18/
drwxr-xr-x  2 zeek zeek     4096 May 20 00:00 2021-05-19/
drwxr-xr-x  2 zeek zeek     4096 May 21 00:00 2021-05-20/
drwxr-xr-x  2 zeek zeek     4096 May 22 00:00 2021-05-21/
drwxr-xr-x  2 zeek zeek     4096 May 23 00:00 2021-05-22/
drwxr-xr-x  2 zeek zeek     4096 May 24 00:00 2021-05-23/
drwxr-xr-x  2 zeek zeek     4096 May 25 00:00 2021-05-24/
drwxr-xr-x  2 zeek zeek     4096 May 26 00:00 2021-05-25/
drwxr-xr-x  2 zeek zeek     4096 May 27 00:00 2021-05-26/
drwxr-xr-x  2 zeek zeek     4096 May 31 14:34 2021-05-27/
drwxr-xr-x  2 zeek zeek     4096 Jun  1 16:37 2021-05-31/
drwxr-xr-x  2 zeek zeek     4096 Jun  7 11:11 2021-06-01/
drwxr-xr-x  2 zeek zeek     4096 Jun  3 00:00 2021-06-02/
drwxr-xr-x  2 zeek zeek     4096 Jun  4 00:00 2021-06-03/
drwxr-xr-x  2 zeek zeek     4096 Jun  5 00:00 2021-06-04/
drwxr-xr-x  2 zeek zeek     4096 Jun  6 00:00 2021-06-05/
drwxr-xr-x  2 zeek zeek     4096 Jun  7 00:00 2021-06-06/
drwxr-xr-x  2 zeek zeek    12288 Jun  9 15:24 2021-06-07/
drwxr-xr-x  2 zeek zeek     4096 Jun  9 00:00 2021-06-08/
drwxr-xr-x  2 zeek zeek    12288 Jun 10 00:00 2021-06-09/
drwxr-xr-x  2 zeek zeek     4096 Jun 11 00:00 2021-06-10/
drwxr-xr-x  2 zeek zeek     4096 Jun 12 00:00 2021-06-11/
drwxr-xr-x  2 zeek zeek     4096 Jun 13 00:00 2021-06-12/
drwxr-xr-x  2 zeek zeek     4096 Jun 14 00:00 2021-06-13/
drwxr-xr-x  2 zeek zeek     4096 Jun 15 00:00 2021-06-14/
drwxr-xr-x  2 zeek zeek     4096 Jun 15 12:00 2021-06-15/
lrwxrwxrwx  1 zeek zeek       22 Jun  9 15:24 current -> /nsm/zeek/spool/zeeksa/
-rw-r--r--  1 root root   110541 Jun 15 12:50 packetloss.log

```
***@***.***:/nsm/zeek/logs/current$ cat stats.log
{"ts":"2021-06-15T14:04:28.317407Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.00209808349609375,"events_proc":349,"events_queued":344,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1544,"active_timers":46,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
{"ts":"2021-06-15T14:09:28.318018Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.0005991458892822266,"events_proc":333,"events_queued":336,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1563,"active_timers":43,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
{"ts":"2021-06-15T14:14:28.318331Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.0008881092071533203,"events_proc":344,"events_queued":340,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1570,"active_timers":47,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
{"ts":"2021-06-15T14:19:28.319335Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.0007231235504150391,"events_proc":332,"events_queued":335,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1561,"active_timers":44,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
{"ts":"2021-06-15T14:24:28.319961Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.0010399818420410157,"events_proc":342,"events_queued":340,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1568,"active_timers":46,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
{"ts":"2021-06-15T14:29:28.320248Z","peer":"zeek","mem":213,"pkts_proc":0,"bytes_recv":0,"pkts_dropped":0,"pkts_link":0,"pkt_lag":0.0006971359252929688,"events_proc":335,"events_queued":337,"active_tcp_conns":0,"active_udp_conns":0,"active_icmp_conns":0,"tcp_conns":0,"udp_conns":0,"icmp_conns":0,"timers":1565,"active_timers":44,"files":0,"active_files":0,"dns_requests":0,"active_dns_requests":0,"reassem_tcp_size":0,"reassem_file_size":0,"reassem_frag_size":0,"reassem_unknown_size":0}
```


***@***.***:/nsm/zeek/logs$ more current

*** current: directory ***

***@***.***:/nsm/zeek/logs$


Do you see alerts generated in /nsm/suricata/eve-$TIMESTAMP.json?

***@***.***:/nsm/zeek/logs$ cd /nsm/suricata
***@***.***:/nsm/suricata$ ll
total 44
drwxr-xr-x  3 suricata socore   4096 Jun 15 11:15 ./
drwxr-xr-x 22 root     root     4096 May 18 16:56 ../
-rw-r--r--  1 suricata suricata   46 Jun  7 19:12 eve-2021-06-07-19:12.json.gz
-rw-r--r--  1 suricata suricata   46 Jun  8 07:02 eve-2021-06-08-07:02.json.gz
-rw-r--r--  1 suricata suricata   46 Jun  9 07:57 eve-2021-06-09-07:57.json.gz
-rw-r--r--  1 suricata suricata   46 Jun  9 14:57 eve-2021-06-09-14:57.json.gz
-rw-r--r--  1 suricata suricata   46 Jun  9 15:24 eve-2021-06-09-15:24.json.gz
-rw-r--r--  1 suricata suricata   46 Jun 10 07:37 eve-2021-06-10-07:37.json.gz
-rw-r--r--  1 suricata suricata   46 Jun 11 07:27 eve-2021-06-11-07:27.json.gz
-rw-r--r--  1 suricata suricata   46 Jun 12 07:17 eve-2021-06-12-07:17.json.gz
-rw-r--r--  1 suricata suricata    0 Jun 15 07:52 eve-2021-06-15-07:52.json
drwxr-xr-x  2 suricata socore   4096 May 18 15:36 extracted/
***@***.***:/nsm/suricata$ more  eve-2021-06-15-07:52.json

***@***.***:/nsm/suricata$ zcat eve-2021-06-12-07:17.json.gz

***@***.***:/nsm/suricata$


If you perform the following command, do you see indices for so-zeek-DATE or so-ids-$DATE?

***@***.***:/nsm/suricata$ so-zeek-
so-zeek-logs     so-zeek-restart  so-zeek-start    so-zeek-stats    so-zeek-stop

so-zeek-logs gives this:

***@***.***D761CE.01D480A0]

so-zeek-restart restarts the services, so-zeek-start or so-zeek-stop does just that start or stops zeek /salt jobs

so-zeek-stats give this:

***@***.***:/nsm/suricata# so-zeek-stats
##############
# Zeek Stats #
##############

Average throughput:

Interface             kpps       mbps       (10s average)
----------------------------------------
localhost/bond0       0.0        0.0

Average packet loss:

     zeeksa: 1623765597.425236 recvd=0 dropped=0 link=0

***@***.***:/nsm/suricata#

I can’t run so-zeek-DATE or so-ids-$DATE

Please advise

Thanks,

Alberto
